### PR TITLE
Cvpdffile/issue83

### PR DESCRIPTION
**Pull request by KavyaReddy732**
# Description
cv
Please provide a short summary explaining what this PR is about.
used params for id which gives dynamical value for the foreign key. 
changed the cvs seeding to retrieve the pdf.
Fixes # (issue)

# How to test?
for testing replace id with some number according to the cv seeding.

Please provide a short summary how your changes can be tested?

# Checklist

- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/CvPdfRender.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React,{useState,useEffect} from 'react';
```
Comment (pdsorensen) [inline src/client/components/CvPdfRender.js:1]: @KavyaReddy732 do you have formatting on? If you do, you can make it automatically happen when saving a file. Here is how: try it out and see if it changes anything: https://stackoverflow.com/questions/39494277/how-do-you-format-code-on-save-in-vs-code

**Review by pdsorensen — CHANGES_REQUESTED** [review]
Just the styles and formatting, otherwise it looks good.

File: src/client/components/CvPdfRender.js (hunk 1 above)
Comment (KavyaReddy732) [reply src/client/components/CvPdfRender.js:1]: i am lost with the styling of iframe. i tried to style but it's not making any changes

File: src/server/seeds/development/02_cvs.js [hunk 2]
```diff
@@ -9,22 +9,22 @@ exports.seed = function(knex) {
           'id': '1',
           'title': 'FrontEnd Developer',
           'description': 'Frontend developer with 3 years experience',
-          'file_url': 'https://zety.com/blog/banking-resume-example',
+          'file_url': 'http://www.africau.edu/images/default/sample.pdf',
```
Comment (pdsorensen) [inline src/server/seeds/development/02_cvs.js:12]: I don't think these needs to be changed?

File: src/client/components/CvPdfRender.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+
+export function CvPdfRender() {
+    const [cvPdfFile, setCvPdfFile] = useState()
+    const { id } = useParams();
+    useEffect(() => {
+        (async () => {
+            const response = await fetch('/api/cv');
```
Comment (pdsorensen) [inline src/client/components/CvPdfRender.js:10]:
```suggestion
            const response = await fetch('/api/cv')
               .json()
               .find(cv => cv.fk_user_id === id);
```

**Review by pdsorensen — CHANGES_REQUESTED** [review]
Almost there.

File: src/server/seeds/development/02_cvs.js (hunk 2 above)
Comment (KavyaReddy732) [reply src/server/seeds/development/02_cvs.js:12]: but, i shared a post in slack before changing it. you and Daniel said ok to change. anyway if you want me to undo the commit then i'll revert it.

Comment (pdsorensen) [reply src/server/seeds/development/02_cvs.js:12]: Ah sorry, that was before i learned of @bytewiz nice trick. :)

Comment (KavyaReddy732) [reply src/server/seeds/development/02_cvs.js:12]: but if i keep original them am getting this
pdf

Comment (pdsorensen) [reply src/server/seeds/development/02_cvs.js:12]: Okay, then it's fine. For future reference, i had to click a little and ended up with this one: 

blob:https://www.wozber.com/21cc7cd6-c62f-4b34-aec4-06d746f3e116

File: src/client/components/CvPdfRender.js (hunk 3 above)
Comment (KavyaReddy732) [reply src/client/components/CvPdfRender.js:10]: if i change the code like this am getting this error (CvPdfRender.js:11 Uncaught (in promise) TypeError: fetch(...).json is not a function)

**Review by pdsorensen — APPROVED** [review]

**Pull request merged**